core/src/main/java/org/apache/druid/data/input/impl/CloudObjectInputSource.java

```diff
@@ -50,6 +52,7 @@ public abstract class CloudObjectInputSource extends AbstractInputSource
   private final List<URI> uris;
   private final List<URI> prefixes;
   private final List<CloudObjectLocation> objects;
+  private final String filter;
```

Comment:
We should mark the field as deprecated and add a small comment suggesting usage of `objectGlob` instead.

Reply:
Thanks for the review @kfaraz, Addressed this suggestion in the latest commit.